vector/src/main/java/im/vector/app/features/settings/VectorPreferences.kt

```diff
@@ -1072,4 +1071,8 @@ class VectorPreferences @Inject constructor(private val context: Context) {
                 .putBoolean(SETTINGS_THREAD_MESSAGES_SYNCED, shouldMigrate)
                 .apply()
     }
+
+    fun isLiveSenderInfo(): Boolean {
+        return defaultPrefs.getBoolean(SETTINGS_LABS_IS_LIVE_SENDER_INFO, false)
```

Comment:
Can you create a boolean pair in `config-settings.xml`? See the doc at the top of this file to see how to do it.

Reply:
Done. Thank you.